src/code/PreRender.c

```diff
@@ -457,27 +461,32 @@ void func_800C213C(PreRender* this, Gfx** gfxP) {
         curRow = 0;
         while (rowsRemaining > 0) {
             s32 uls = 0;
+            s32 ult = curRow;
             s32 lrs = this->width - 1;
-            s32 ult;
             s32 lrt;
 
             // Make sure that we don't load past the end of the source image
-            nRows = MIN(rowsRemaining, nRows);
+            if (nRows > rowsRemaining) {
+                nRows = rowsRemaining;
+            }
 
-            // Determine the upper and lower bounds of the rect to draw
-            ult = curRow;
+            // Determine the lower bound of the rect to draw
             lrt = curRow + nRows - 1;
 
             // Load the frame buffer line
             gDPLoadMultiTile(gfx++, this->fbufSave, 0x0000, G_TX_RENDERTILE, G_IM_FMT_RGBA, G_IM_SIZ_16b, this->width,
                              this->height, uls, ult, lrs, lrt, 0, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMIRROR | G_TX_WRAP,
                              G_TX_NOMASK, G_TX_NOMASK, G_TX_NOLOD, G_TX_NOLOD);
 
+            rtile = rtile; // fake
+
```

Comment:
seems more common in the codebase to comment (fake match?) for things, but it will come up in a ctrl f for fake anyway